extras/openshift/templates/heketi-template.json

```diff
@@ -150,6 +150,10 @@
                   {
                     "name": "db",
                     "mountPath": "/var/lib/heketi"
+                  },
+                  {
+                    "name": "db",
```

Comment:
name should match name in ["volumes"](https://github.com/heketi/heketi/pull/619/files#diff-db4f124381d5c61236daeb291895e5f6R177). We can have a separate volumes "entry" for the same gluster(or heketidbstorage) volume, which internally will mount the same gluster(or heketidbstorage) volume on the host and the bind mount it for the different volume entry(which I feel unnecessary). I will test this out right away. I feel heketi container should be as light as possible.